### PR TITLE
Fix to update Hub ConfigMap values

### PR DESCRIPTION
**Pull request by pratap0007**
This patch fixes to update ConfigMap's values and
whenever corresponding keys gets change in the Hub CR

Signed-off-by: Shiv Verma <shverma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: pkg/reconciler/kubernetes/tektonhub/tektonhub.go [hunk 1]
```diff
@@ -488,32 +500,6 @@ func (r *Reconciler) validateApiDependencies(ctx context.Context, th *v1alpha1.T
 			return err
 		}
 	}
-
-	_, err = r.getConfigMap(ctx, api, namespace, apiConfigMapKeys)
```
Comment (piyush-garg) [inline pkg/reconciler/kubernetes/tektonhub/tektonhub.go:492]: will upgrade work fine here? will the old configmap will be removed.

File: pkg/reconciler/kubernetes/tektonhub/tektonhub.go [hunk 2]
```diff
@@ -1092,3 +1063,63 @@ func (r *Reconciler) targetNamespaceCheck(ctx context.Context, th *v1alpha1.Tekt
 	}
 	return nil
 }
+
+// Update the ConfigMap values with CR values
+func updateConfigMapValues(configMapName string, prop interface{}) mf.Transformer {
```
Comment (piyush-garg) [inline pkg/reconciler/kubernetes/tektonhub/tektonhub.go:1068]: this is mostly duplicate func

File: pkg/reconciler/kubernetes/tektonhub/tektonhub.go (hunk 1 above)
Comment (pratap0007) [reply pkg/reconciler/kubernetes/tektonhub/tektonhub.go:492]: yes and  it's removing

**Comment by pratap0007** [comment]
Have tested this patch with new installation and upgrade, and working fine!

**Pull request merged**